src/il_representations/scripts/joint_training.py

```diff
@@ -326,10 +368,109 @@ def learn_repl_bc(repl_learner, repl_datasets, bc_learner, bc_augmentation_fn,
     # return final saved policy path
     pol_path = save_dir / "policy_final.ckpt"
     os.makedirs(save_dir, exist_ok=True)
-    th.save(bc_learner.policy, pol_path)
+    torch.save(bc_learner.policy, pol_path)
     return pol_path
 
 
+@train_ex.capture
+def learn_repl_gail(repl_learner, repl_datasets, bc_learner,
```

Comment:
Are there specific reasons why you commented out the whole method? Would you prefer to uncomment it or just delete it?

Reply:
Deleted function along with other GAIL joint train stuff